addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -47,9 +47,9 @@
                 <div t-attf-id="collapseDependencies{{ dependency_index }}" class="collapse" aria-expanded="false">
                     <ul>
                         <li t-foreach="dependency_value" t-as="item" t-key="item_index">
-                            <t t-set="link_text">This URL is contained in the '%s' of the following '%s'</t>
+                            <t t-set="link_text">This URL is contained in the “%(field)s” of the following “%(model)s”</t>
                             <a t-att-href="item.link" target="_blank">
-                                <t t-out="sprintf(link_text, item.field_name, item.model_name)"/>: <b t-out="item.record_name"/>
+                                <t t-out="sprintf(link_text, { field: item.field_name, model: item.model_name })"/>: <b t-out="item.record_name"/>
```

Comment:
Note that there is pretty much the same problem with the two sides of the colon separator, but at least it's not a sentence.